stdlib/arg.ml

```diff
@@ -304,20 +304,32 @@ let parse_expand l f msg =
 
 let second_word s =
   let len = String.length s in
-  let rec loop n =
-    if n >= len then len
-    else if s.[n] = ' ' then loop (n+1)
-    else n
+  let rec loop sep n =
+    if n >= len then sep, len
+    else if s.[n] = sep then loop sep (n+1)
+    else sep, n
   in
-  try loop (String.index s ' ')
-  with Not_found -> len
+  match String.index s '\t' with
+  | n -> loop '\t' n
```

Comment:
Shouldn't we allow whitespaces after the first \t character (and not extra \t characters)?  I think allowing these extra whitespaces is to allow some graphical alignment in the source code, and this would be achieves by whitespaces even if \t is used as the separator, no?